.github/workflows/verify-smart-contracts.yaml

```diff
@@ -6,8 +6,6 @@ on:
       - main
       - staging
       - development
-    paths:
-      - "packages/contracts/**"
```

Comment:
why paths are removed?

Reply:
I added requirement to pass "Verify **" actions for pull requests, but github not support conditional actions. So need remove pass to keep them as required